donkeycar/parts/image.py

```diff
@@ -104,3 +108,11 @@ def run(self, img_arr):
 
     def shutdown(self):
         pass
+
+
+class ImgPrecondition:
```

Comment:
I want to get rid of this and fold this into either the camera or a new threaded part.

Reply:
I can make this part threaded, that's easy, but it's very fast, so won't make a big difference.

I don't think it should go into the camera which returns uint8 native images. These also go into the tub, etc, so there we don't want to convert back again. The place where normalised floats are required are in the pilot. I can put this normalisation/float conversion into a pilot base class and have that virtually dispatch `run` into derived parts.

Reply:
I meant the `normalize_and _crop` won't actually be doing much given we are getting rid of `ROI_CROP_*`.  The only thing that its doing is converting an uin8 image to a normalized image.

Reply:
Yes correct, and we need both representations, the uint8 for the tub and the normalised float for the pilot. I think this should go into the pilot, that makes the most sense to me.